src/Umbraco.Web/Models/Mapping/SensitivePropertiesResolver.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using AutoMapper;
+using Umbraco.Core.Models;
+using Umbraco.Core.Services;
+using Umbraco.Web.Models.ContentEditing;
+
+namespace Umbraco.Web.Models.Mapping
+{
+    internal class SensitivePropertiesResolver<TSource> : IValueResolver where TSource : IContentBase
```

Comment:
This class doesn't seem to be required, this logic should just exist on the MemberSensitivePropertiesResolver since that's the only place it's used.

Reply:
I think this was done because the functionality and inheritance of the `TabsAndPropertiesResolver` was copied. There's just no need for a base class if there's never going to be a single item that inherits from it.